sw/device/silicon_creator/lib/error.h

```diff
@@ -156,7 +157,11 @@ enum module_ {
   /* The high-byte of kErrorInterrupt is modified with the interrupt cause */ \
   X(kErrorRomExtInterrupt,            ERROR_(0, kModuleRomExtInterrupt, kUnknown)), \
   \
-  X(kErrorBootLogInvalid,             ERROR_(1, kModuleBootLog, kInternal))
+  X(kErrorBootLogInvalid,             ERROR_(1, kModuleBootLog, kInternal)), \
+  \
+  X(kErrorAsn1Internal,               ERROR_(1, kModuleAsn1, kInternal)), \
+  X(kErrorAsn1InvalidArgument,        ERROR_(2, kModuleAsn1, kInvalidArgument)), \
+  X(kErrorAsn1BufferExhausted,        ERROR_(3, kModuleAsn1, kResourceExhausted))
```

Comment:
Should we add a trailing \ so that we get better diffs?

Reply:
I tried that, and also add a trailing ",". But clang-format will not let me do that, even though it is supposedly disable for that block of code...